ui/src/views/Hooks/ListHooks/index.jsx

```diff
@@ -89,7 +93,11 @@ export default class ListHooks extends Component {
             softSearch
             tree={tree}
             onEmptySearch={
-              <Typography variant="subtitle1">
+              <Typography
+                variant="subtitle1"
+                placeholder="Updated Hook contains"
+                color="inherit"
+                onClick={this.handleHookUpdateSearch}>
                 No items for search term {hookSearch}
```

Comment:
This text looks different from what #3044 shows. A bit of exploring the git history shows this issue already got fixed, in #2779.  So, maybe there's nothing to change here?

Reply:
So where I am a bit that where could I change as I thought it was about to enhance the empty state by using the color component in the typography theme.

Reply:
Nope, the expected behavior doesn't say anything about color:
>  If there are no hooks to display, it should say something other than "No items for search term".

Reply:
Oh, I've misunderstood the issue, too!

I'll update the issue itself with a better explanation.

Reply:
ok.